asm/istio/istio-operator.yaml

```
@@ -104,3 +104,7 @@ spec:
         clusterName: "cn-PROJECTID-us-central1-c-asm-cluster" # {"$ref":"#/definitions/io.k8s.cli.substitutions.cluster-name"}
       meshID: "proj-PROJECT_NUMBER" # {"$ref":"#/definitions/io.k8s.cli.substitutions.mesh-id"}
       network: "NETWORK_ID" # {"$ref":"#/definitions/io.k8s.cli.setters.gcloud.compute.network"}
+    gateways:
+      istio-ingressgateway:
+        # Enable gateway injection
+        injectionTemplate: gateway
```

```suggestion
    gateways:
      istio-ingressgateway:
        # Enable gateway injection
        injectionTemplate: gateway      
      istio-egressgateway:
        # Enable gateway injection
        injectionTemplate: gateway
```

so it applies to both